helmfile/values/podsecuritypolicies/service/opensearch.yaml.gotmpl

```diff
@@ -0,0 +1,38 @@
+constraints:
+  opensearch-system:
+    opensearch-nodes:
+      podSelectorLabels:
+        app.kubernetes.io/name: opensearch
+      allow:
+        allowPrivilegeEscalation: true
+        # allowedUnsafeSysctls:
+        #   - vm.max_map_count
```

Comment:
Not implemented?

Reply:
Not as of yet, I want to revisit it once we are fully off Kubernetes PSPs as it will probably be simpler to manage it then.